docs/dev/Lua API.rst

```diff
@@ -953,6 +953,10 @@ can be omitted.
 
   Convert a ``df.language_name`` (or only the last name part) to string.
 
+* ``dfhack.GenerateName(name,language,type,major_selector,minor_selector)``
```

Comment:
Would it be feasible to provide an additional signature of `dfhack.GenerateName(name, language, type)` that chooses appropriate selectors based on the `language_name_type`? I can write the LuaApi glue if you write the cpp.

Reply:
There is no single set of rules for how the major/minor selectors are chosen - sometimes they're taken from the global list, and sometimes some are taken from the entity definition.